test/files/neg/t11575b.check

```diff
@@ -0,0 +1,7 @@
+LocalImpl.scala:2: error: incompatible type in overriding
+private[package <empty>] var counts: Array[Int] (defined in class Cover);
+ found   : String
+ required: Array[Int]
+    override var counts: String = ""
+                 ^
```

Comment:
This tests the status quo, but I'm not sure that it's even sensible to let Scala vars claim to override Java fields.